posts/ml-sagemaker.md

```diff
@@ -0,0 +1,201 @@
+---
+authors:
+- Mallik Sambaraju
+tags:
+- Machine Learning
+- Data Science
+- AWS
+- Sagemaker
+- Python
+date: 2019-03-22T12:24:13.000Z
+title: "Linear Regression using AWS Sagemaker"
+image: 
+---
+
+Various industries such as Banking, Manufacturing, Insurance, Health, and Defence are using Machine Learning to solve many use cases such as Fraud Detection, Improving Healthcare, Personal security, Providing more secured transportation e.t.c. Recently [Ippon Technologies](https://en.ippon.tech) sent me to a summit on Data Science where I learnt about how organizations are using Machine Learning to improve their business and productivity. During the course of the summit many service and cloud providers presented various tools, libraries, algorithms and platforms for developing Machine Learning applications. One of the services introduced to me was Sagemaker by AWS. In this blog I will talk about How I implemented a basic regression model.
+
+# Machine Learning Process
+Typically Machine Learning process is an iterative process, it starts with identifying a use case to solve. Some of the steps involved in this process are as follows. This could change based on the use case you are trying to solve.
```

Comment:
```suggestion
Typically Machine Learning process is an iterative process; it starts with identifying a use case to solve. Some of the steps involved in this process are as follows. This could change based on the use case you are trying to solve.
```